docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -315,7 +315,11 @@ Note that the diagnostics bundle is intended for debugging purposes only, its st
 IMPORTANT: The configuration files in the archive may have credentials in *plain text*.
 These will need to be manually redacted before the archive is shared.
 
-The diagnostics command is also able to collect `pprof` data in the archive if the `--pprof` flag is passed:
+The diagnostics command is also able to collect `pprof` data in the archive if the `--pprof` flag is added when collecting the diagnostic.
+
+To collect the `pprof` data:
+- Enable/add `agent.monitoring.pprof.enabled: false` to `elastic-agent.yml`
```

Comment:
Are you sure this should be false? I believe it should be true
```suggestion
- Enable/add `agent.monitoring.pprof.enabled: true` to `elastic-agent.yml`
```

Reply:
🤦 Thank you

Reply:
Too much copy paste